packages/vuetify/src/services/theme/index.ts

```diff
@@ -179,6 +179,22 @@ export class Theme extends Service {
     this.vueMeta = true
 
     const meta = root.$meta()
+
+    const metaAdd = typeof meta.addApp === 'function'
+    if (metaAdd) {
+      const { set } = root.$meta().addApp('vuetify')
```

Comment:
This should work too I guess:

```suggestion
      const { set } = meta.addApp('vuetify')
```